pkg/odo/cli/preference/view.go

```diff
@@ -46,7 +45,7 @@ func (o *ViewOptions) Run(cmd *cobra.Command) (err error) {
 	cfg, err := preference.New()
 
 	if err != nil {
-		util.LogErrorAndExit(err, "")
```

Comment:
I think we should keep this as is

Reply:
I think, we should add log before return. But not sure why we want to keep this `LogErrorAndExit`. Can you help me understand how this will be helpful